pkg/getter/httpgetter.go

```diff
@@ -34,7 +35,15 @@ type httpGetter struct {
 func (g *httpGetter) Get(href string) (*bytes.Buffer, error) {
 	buf := bytes.NewBuffer(nil)
 
-	resp, err := g.client.Get(href)
+	// Set a helm specific user agent so that a repo server and metrics can
+	// separate helm calls from other tools interacting with repos.
+	req, err := http.NewRequest("GET", href, nil)
+	if err != nil {
+		return buf, err
+	}
+	req.Header.Set("User-Agent", "Kubernetes_Helm/"+version.GetVersion())
```

Comment:
Can we remove the `Kubernetes_` part from the user agent string? Direct forks of Helm should all be able to send the same user string, so this might be a little too granular. Tools other than Helm would send differing user agents ofc e.g. `Monocular/0.6`, but forks like `myorg/helm` should still be able to send `Helm/2.7` and the server should be able to handle the request.

I wonder if the `v` should be removed from the version string as well as it seems like the `v` is omitted in Mozilla's User-Agent string. That's more of an open question, though. I'm curious to hear your opinion on this. :)

Reply:
> I wonder if the v should be removed from the version string as well as it seems like the v is omitted in Mozilla's User-Agent string.

I can remove the `v` from the user agent. That makes sense.

> Can we remove the Kubernetes_ part from the user agent string?

Sure. I even poked around other tools in Kubernetes and didn't see that part.